tools/boards.txt.py

```diff
@@ -1436,6 +1462,8 @@ def all_flash_map ():
     f1m.update( flash_map(    1024,     512 ))
     f1m.update( flash_map(    1024))
 
+    f2m0.update(flash_map(  2*1024))
```

Comment:
This is not needed.  The 2M already has a no-FS option.  Please remove.

Reply:
This "new" flash-size option allows to hide FS options. Only 2M without FS size appear in the menu.

Reply:
I have modified, can you check if everything is ok?